src/utils/sosmeta-utils.ts

```diff
@@ -153,6 +153,22 @@ namespace SosmetaUtils {
     }
   };
 
+  /**
+   * Handles creation of MetaformFields from Sosmeta Schema Properties of type boolean
+   *
+   * @param sosmetaSection
+   * @returns MetaformField
+   */
+  const handleSosmetaBooleanSection = (sosmetaSection: any) => {
+    try {
+      const { required } = sosmetaSection;
+
+      return [ handleSosmetaBooleanField(sosmetaSection.sosmeta.name[0].value, sosmetaSection, required) ];
+    } catch (e) {
+      // TODO: Error handling/messages
```

Comment:
If trycatch is needed here, do proper error handling.

Reply:
Try catch removed and found any errors were still being handled in the convertSosmetaToMetaform function.